palvelutarjotin/settings.py

```diff
@@ -337,6 +338,8 @@
 
 MAX_UPLOAD_SIZE = 2621440  # 2MB
 
+PERSONAL_DATA_RETENTION_PERIOD_MONTHS = env("PERSONAL_DATA_RETENTION_PERIOD_MONTHS")
```

Comment:
it would be a good practise to add these to `.env.example` -file and to `docker-compose.env.yaml.example`, but it seems like those does not include earlier params either. Also the CI configuration is not able to read this at the moment. But, I think this is fine like this, because I don't think we really change the value so often.

Reply:
Yeah I agree. The idea was that most probably this value doesn't need to be changed ever, so it could be hard-coded, but because it needs practically zero effort to make it also env configurable, added that as well 🙂